src/services/ckan/datasetCount.ts

```diff
@@ -1,14 +1,16 @@
 // SPDX-FileCopyrightText: 2024 PNED G.I.E.
+// SDPX-Contributor: Stichting Health-RI
 //
 // SPDX-License-Identifier: Apache-2.0
 
-import axios from 'axios';
+// import axios from 'axios';
 import { constructCkanActionUrl } from './utils';
 
 export const makeDatasetCount = (DMS: string) => {
   return async (): Promise<number> => {
     const url = constructCkanActionUrl(DMS, 'dataset_list');
-    const response = await axios.get(url);
-    return response.data.result.count;
+    const raw_response = await fetch(url, { cache: 'force-cache', next: { revalidate: 24 * 3600 } });
```

Comment:
important: move all literals `24 * 3600` to environment variable configuration, like `CKAN_BACKEND_CACHE_TTL` or something more readable, and default it to 24h.

I don't know Next.js enough, but you should pay attention that this env variable may be accessible on client and server rendereded components.

Reply:
I have added a const in top of the file. We think an environment variable is too verbose? Agree?

Reply:
I was thinking on somewhere central place and reuse a single variable on both pleaces. But indeed perhaps it's not needed now.